explorer/src/components/account/BlockHistoryCard.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { TableCardBody } from "components/common/TableCardBody";
+import { useBlock, useFetchBlock } from "providers/accounts/block";
+import { Signature } from "components/common/Signature";
+
+export function BlockHistoryCard({
+  slot,
+  blockhash,
+}: {
+  slot: string;
+  blockhash: string;
+}) {
+  const block = useBlock(slot ? slot : blockhash);
+  const fetchBlock = useFetchBlock();
+  const refresh = () =>
+    fetchBlock(slot ? slot : blockhash, slot ? "slot" : "blockhash");
+
+  React.useEffect(() => {
+    if (!block) refresh();
+  }, [slot, blockhash, block]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  function invalidOverview() {
+    return (
+      <tr>
+        <td>ERROR</td>
+        <td>Could not retrieve block details</td>
+      </tr>
+    );
+  }
+
+  function invalidTransactions() {
+    return (
+      <tr>
+        <td>ERROR</td>
+        <td>Could not retrieve transactions</td>
+      </tr>
+    );
+  }
+
+  return (
+    <>
+      <div className="card">
+        <div className="card-header">
+          <h3 className="card-header-title mb-0 d-flex align-items-center">
+            Overview
+          </h3>
+        </div>
+
+        <TableCardBody>
+          {block
+            ? block.data?.Tags
+              ? block.data?.Tags.filter(
+                  (tag: any) =>
+                    tag.name === "slot" ||
+                    tag.name === "parentSlot" ||
+                    tag.name === "blockhash"
+                ).map((tag: any) => {
+                  return (
+                    <tr>
+                      <td>{tag.name}</td>
+                      <td className="text-lg-right">{tag.value}</td>
```

Comment:
Parent slot value should be "linkified" for easy navigation. In fact, it would be great if we create a `BlockSlot` component (similar to `Address`) which has some options for turning slot numbers into links.

Reply:
Ohhhh I like this idea a lot!
Thanks, will add that in.